pkg/bootstrapper/kubelet.go

```diff
@@ -161,3 +179,76 @@ func (k *kubeletService) setRecoveryActions() error {
 	}
 	return nil
 }
+
+// startService is a helper to start a given service
+func startService(serviceObj *mgr.Service) error {
+	if serviceObj == nil {
+		return fmt.Errorf("service object should not be nil")
+	}
+	isServiceRunning, err := isServiceRunning(serviceObj)
+	if err != nil {
+		return fmt.Errorf("unable to check if service is running: %v", err)
+	}
+	if !isServiceRunning {
+		err := serviceObj.Start()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// controlService is a helper to send control signal to a given service
+func controlService(serviceObj *mgr.Service, cmd svc.Cmd, desiredState svc.State) error {
+	if serviceObj == nil {
+		return fmt.Errorf("service object should not be nil")
+	}
+	status, err := serviceObj.Control(cmd)
+	if err != nil {
+		return err
+	}
+	// Most of the rest of the function borrowed from the package (golang.org/x/sys/windows/svc/mgr) example
+	// Arbitrary wait time
```

Comment:
Please mention what that arbitrary wait time is.

Reply:
done